back-office/shop-parameters/customer-settings/titles.md

```diff
@@ -0,0 +1,58 @@
+# **SPECIFICATIONS - CUSTOMERS SETTINGS > TITLES PAGE**
+
+
+_As a merchant, I want to be able to define my customers._
+
+## Listing
+
+As usual, four tables options are displayed on the top-right hand corner as icons: 
+
+- Add new<br>
+- Refresh list<br>
+- Show SQL query<br>
+- Export to SQL manager<br>
+
+By default, the table is divided into the following columns:
+
+- ID<br>
+- Social title<br>
+- Gender<br>
+- Image<br>
+
+Searching fields are available for users to narrow down their search. 
+
+At the beginning of each line is a checkbox while, at the end of each line, two actions can be selected: _Edit_ and _Delete_. Checkboxes play with the 'Bulk actions' selector that allows either to 'Select all', 'Unselect all', or 'Delete selected'.
```

Comment:
```suggestion
At the beginning of each line is a checkbox while, at the end of each line, two actions can be done: _Edit_ and _Delete_. The main action is _Edit_ and it can be performed by clicking on the line.
Checkboxes play with the 'Bulk actions' selector that allows either to 'Select all', 'Unselect all', or 'Delete selected'.
```

Reply:
We should specify the multistore behavior here:
As the add title page doesn’t have shop association block, when the user add a title it's added for all shops, no matter the context selected
So, all titles are listed here no matter the context selected (all shops, a group or a specific shop)